app/src/main/java/cm/aptoide/pt/install/installer/DefaultInstaller.java

```diff
@@ -110,9 +110,11 @@ public PackageManager getPackageManager() {
             .first()
             .flatMap(isInstalled -> {
               if (isInstalled) {
-                installation.setStatus(Installed.STATUS_COMPLETED);
-                installation.save();
-                return Observable.just(null);
+                installation.setStatus(RoomInstalled.STATUS_COMPLETED);
+                return installation.save()
+                    .toObservable()
+                    .map(__ -> null);
```

Comment:
I don't think there is a problem passing the installation instead of null here, can you confirm?